### DISCUSSION TITLE
Start - sub flow (on different lane) -> receive back the response

### DISCUSSION THREAD

**Discussion by bsdgithub** [flowsforapex/apex-flowsforapex#763] [Q&A]
Hello,
I'm a little bit lost between lanes :-)
My example is on 23.X.
I have a human task that can trigger a subflow on a different lane (but still remain on current human task) and maybe another on another lane and wait for the subflows to send back the result. After that the user can continue the flow. I was trying to use signal throwing and receiving events, but I've failed.
Could you guide me?

**Comment by rallen2010** [comment]
email me - Richard  at. Flowquest.net
